src/screens/dealsFeed/dealCard/__tests__/__snapshots__/dealCard.test.tsx.snap.android

```diff
@@ -0,0 +1,86 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I'm currently unsure about the usefulness of snapshot tests. I'd much prefer tests to assert the state of the rendered output, rather than relying on humans to decide "this snapshot change looks right/wrong". But for now I've included them as an example of unit tests.